include/core/parameters_multiphysics.h

```diff
@@ -47,6 +47,17 @@ namespace Parameters
     adaptive
   };
 
+  /**
+   * @brief Different transomation function types for the signed distance:
+   *  - tanh: hyperbolic tangent
+   *  - piecewise_polynomial: 4th degree piecewise polynomial
```

Comment:
```suggestion
   * @brief Different transformation function types for the signed distance:
   *  - tanh: hyperbolic tangent
   *  - piecewise_polynomial: 4th degree piecewise polynomial
```

Could you please give the equation or the form of the polynomial